frontend/src/tests/components/eMIB/EditEmailActionDialog.test.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { shallow } from "enzyme";
+import EditEmailActionDialog, {
+  EDIT_MODE,
+  ACTION_TYPE
+} from "../../../components/eMIB/EditEmailActionDialog";
+
```

Comment:
Can you please add some describe block here. One for **Email** and one for **Task**.
I think it'll be clearer.

Reply:
Done!